### PR TITLE
User/andreww msft010

### PR DESCRIPTION
**Pull request by andreww-msft**
Fixed quarantined samples (maybe)

### CODE REVIEW

File: Samples/nuget.config [hunk 1]
```diff
@@ -10,6 +10,8 @@
     <packageSources>
     <clear />
         <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+        <add key="Internal" value="https://pkgs.dev.azure.com/microsoft/ProjectReunion/_packaging/Project.Reunion.nuget.internal/nuget/v3/index.json" />
+        <add key="Local" value="C:\_Data\Cobalt\Reunion" />
```
Comment (kythant) [inline Samples/nuget.config:14]: Seems like this line is causing issues in the pipeline

Comment (erik0686) [reply Samples/nuget.config:14]: Also, I think we should not be using the internal feed :)

**Comment by erik0686** [comment]
Can we change the title of the PR? Other than that this looks good to me :)

**Review by erik0686 — APPROVED** [review]

**Comment by erik0686** [comment]
Should there be a follow up PR to move this sample from the quarantine folder?

**Comment by kythant** [comment]
Some of these Samples are still in the quarantined folder. We need to move them to the Samples folder so that they can be properly tested by the CI/PR pipeline.

**Pull request merged**